lib/ansible/plugins/lookup/cyberarkpassword.py

```diff
@@ -0,0 +1,124 @@
+
+
+#
+#
+#
+#
+# USAGE: {{lookup('cyberarkpassword', AppID='Application1', Query='safe=Safe1;Folder=root;Object=User1',
+#                 Output='Password,PassProps.UserName,PassProps.Address' [, extraParms])}}
+#
+# It Requires CyberArk AIM Installed, and /opt/CARKaim/sdk/clipasswordsdk
+#
+#  Parameter names AppID, Query, Output are case sensitive.
+#
+#  Args:
+#      AppID (str): Defines the unique ID of the application that is issuing the password request.
+#      Query (str): Describes the filter criteria for the password retrieval.
+#      Output (str): Specifies the desired output fields separated by commas. They could be: Password, PassProps.<property>, PasswordChangeInProcess
+#      Optionally, you can specify extra parameters recognized by clipasswordsdk (like FailRequestOnPasswordChange, Queryformat, Reason, etc.)
+#
+#  Returns:
+#      dict: A dictionary with 'password' as key for the credential, passprops.<property>, passwordchangeinprocess
+#            If the specified property does not exist for this password, the value <na> will be returned for this property.
+#            If the value of the specified property is empty, <null> will be returned.
+#
+#
+# for extraParms values please check parameters for clipasswordsdk in CyberArk's "Credential Provider and ASCP Implementation Guide"
+#
+# For Ansible on windows, please change the -parameters (-p, -d, and -o) to /parameters (/p, /d, and /o) and change the location of CLIPasswordSDK.exe
+#
+#
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import subprocess
+from ansible.errors import AnsibleError
```

Comment:
Purestyle note: add a blank line between python stdlib imports and ansible imports